control-plane/csi-driver/src/bin/node/main_.rs

```diff
@@ -93,10 +94,26 @@ impl AsyncWrite for UnixStream {
 const GRPC_PORT: u16 = 50051;
 
 pub(super) async fn main() -> anyhow::Result<()> {
+    // Parse command line arguments.
     let matches = clap::Command::new(utils::package_description!())
         .about("k8s sidecar for IoEngine implementing CSI among others")
         .version(utils::version_info_str!())
         .subcommand_negates_reqs(true)
+        .arg(
+            Arg::new("rest-endpoint")
+                .long("rest-endpoint")
+                .env("ENDPOINT")
+                .default_value("http://ksnode-1:30011")
```

Comment:
Should we make this registration optional? Example: if not enabled we don't start these bits
This will allow for us to still test csi-node in isolation, otherwise we're making rest a requirement

Reply:
Yeah we can, but from an end user perspective that option would be not useful. Maybe we can add it when we want to have it.

Reply:
Not for the users, for our own testing, example we test csi-node in isolation

Reply:
Yeah, we can any naming suggestions for the cli arg?

Reply:
Done.

Reply:
Rest should be optional, so no default value 

```suggestion
```